src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -28,6 +28,26 @@ export function getStaticProps(context) {
   };
 }
 
+<InlineFilter filters={['flutter']}>
+## Prerequisites
+
+* An app set up according to the [getting started walkthrough](/[platform]/build-a-backend/auth/set-up-auth/)
+
+</InlineFilter>
+
+<InlineFilter filters={['android']}>
+## Prerequisites
+
+* An app set up according to the [getting started walkthrough](/[platform]/build-a-backend/auth/set-up-auth/)
+
+</InlineFilter>
+
+<InlineFilter filters={['swift']}>
+## Prerequisites
+
+* An app set up according to the [getting started walkthrough](/[platform]/build-a-backend/auth/set-up-auth/)
+
+</InlineFilter>
```

Comment:
These look like the same links and text? It should be possible to just have one setup for all three of these filters like so if these links are supposed to be the same:

```suggestion
<InlineFilter filters={['flutter', 'android', 'swift']}>
## Prerequisites

* An app set up according to the [getting started walkthrough](/[platform]/build-a-backend/auth/set-up-auth/)

</InlineFilter>
```